dired_misc.py

```diff
@@ -112,23 +112,19 @@ def run(self, edit, preview=True):
         files = self.get_marked() or self.get_selected(parent=False)
         if not files:
             return sublime.status_message('Nothing chosen')
-        if OSX and preview:
-            cmd = ["qlmanage", "-p"]
-            for filename in files:
-                fqn = join(self.path, filename)
-                cmd.append(fqn)
-            subprocess.call(cmd)
-        else:
-            if OSX:
-                launch = lambda f: subprocess.call(['open', f])
-            elif NT:
-                # the "" before filename is a trick for batch files and such
-                launch = lambda f: subprocess.call('start "" "%s"' % f, shell=True)
+        if OSX:
+            if preview:
+                launch = lambda f: subprocess.call(['qlmanage', '-p', f], cwd=self.path)
```

Comment:
Did you test this? I have no OSX, so I’m not sure because previously there was one call of `qlmanage` with all paths as arguments.
This change makes call for each file separately.
@aziz ?

In previous comment I meant only case when preview is false.